docx-asciidoc-conversion/README.md

```diff
@@ -16,39 +17,42 @@ The procedure was tested using the INSPIRE Metadata Technical Guidance 2.0. The
 
 ### Conversion Procedure
 
-1. Correct the original docx document, save as `md_2_fixed.docx`:
-   - apply *XML Example* style to all XML examples (about 7 cases were found in `md_2_0_1.docx` where this fix was required)
-   - remove paragraph breaks from inside footnotes (two cases were found in `md_2_0_1.docx` where this fix was required)
+1. Create a new folder with the name of the document to convert:
+   ```shell
+    mkdir my_example_data_specifications_EX
+    ```
 
-2. Convert the corrected docx document to *AsciiDoc* using Pandoc, save as `md_2_pandoc.adoc`, extract images to `media` folder:
+2. Copy the Microsoft Word document into the new folder. Convert the Microsoft Word document into the last .docx format:
     ```shell
-    pandoc md_2_fixed.docx -f docx -t asciidoc -o md_2_pandoc.adoc --wrap=none --extract-media=. --markdown-headings=atx
+    From Microsoft Word document: File -> Informations -> Convert
     ```
     
-1. Convert the corrected docx document to *Markdown* using Pandoc, enabling the [styles extension](https://pandoc.org/MANUAL.html#ext-styles), save as `md_2_styles.md`:
+3. Launch the preliminary folder script from main folder. This script contains instructions for the first conversion to asciidoc.:
     ```shell
-    pandoc md_2_fixed.docx -f docx+styles -t markdown -o md_2_styles.md --wrap=none --markdown-headings=atx
+    .\pandoc-folder.bat .\my_example_data_specifications_EX
     ```
     
-4. Run the `post_pandoc.py` Python script with 3 arguments: adoc file from 2., md file from 3. and `attributes.adoc` file, it will create `md_2_final.adoc` file:
+4. Launch the conversion script from the main folder, using the correct data format between *metadata*, *services* and *data*. In fact, if a specific modification is required for a type of document, it will be possible to specifically modify one of the three libraries:
     ```shell
-    python post_pandoc.py md_2_pandoc.adoc md_2_styles.md attributes.adoc
+    .\pandoc-postpy.bat .\my_example_data_specifications_EX data
     ```
     
-    The script applies AsciiDoc admonitions to TG Requirement, TG Recommendation and Conformance Class sections, source blocks to XML Examples and blockquotes to quote sections of `md_2_pandoc.adoc` based on additional style information included in`md_2_styles.md`. It also corrects initial (unnumbered) section headings, removes manual TOC entries, inserts automatic TOC. AsciiDoc does not yet support custom section numbering (see [this issue](https://github.com/asciidoctor/asciidoctor/issues/3311)), therefore 'Annex X' text is inserted in relevant headings by the script.
+    The script applies AsciiDoc admonitions to TG Requirement, TG Recommendation and Conformance Class sections, source blocks to XML Examples and blockquotes to quote sections of  md_2_pandoc.adoc based on additional style information included in md_2_styles.md. 
```

Comment:
instead of 'admonitions', maybe 'icons' (or something similar such as 'graphics' or 'emoji').

Reply:
Better 'formatting'